src/renderer/components/NewProjectModal/NewProjectView.tsx

```diff
@@ -45,24 +46,31 @@ const NewProjectView = ({
   setProjectName,
 }: Props) => {
   const dispatch = useDispatch();
+  const [isAwaitingProjectName, setIsAwaitingProjectName] = useState(true);
 
-  const setProjectInStore = async (project: RuntimeProject) => {
-    dispatch(projectCreated(project));
-  };
+  const setProjectInStore = useCallback(
+    async (project: RuntimeProject) => {
+      dispatch(projectCreated(project));
+    },
+    [dispatch]
+  );
 
-  const handleContinue = async () => {
+  const handleContinue = useCallback(async () => {
     const project = await makeProjectWithoutMedia(projectName);
     if (project === null) {
       return;
     }
     setProjectInStore(project);
     nextView();
-  };
+  }, [nextView, projectName, setProjectInStore]);
+
+  useKeypress(handleContinue, !isAwaitingProjectName, ['Enter', 'NumpadEnter']);
```

Comment:
You'll also need to use !projectName.trim() here, because at the moment when you go forward then back to this page it doesn't let you use the enter key until you've changed the value in the project name field

Reply:
Hey Emma here is the change: [9033c84](https://github.com/chloebrett/mlvet/pull/175/commits/9033c84494d7d467e54bf2019bb9095aa396cb7c)

I was not able to use !projectName.trim() since the useKeypress hook no longer negates the useKeypressEnabled condition.

And I had to add !== '' since typescript was not happy when I passed a string instead of boolean variable.